Week2/homeWORK/2_eventOddREPORTER.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+//let numberReporter = 20;
+
+for (let i = 1; i <= 20; i++) {
+    if (i % 2 == 0) {
+        console.log('The number is '+i+' even');
+    }else{
```

Comment:
```suggestion
    } else {
```
Careful with spaces 👍